enterprise/frontend/src/metabase-enterprise/metabot/hooks.ts

```diff
@@ -30,16 +38,21 @@ export const useMetabotAgent = () => {
   });
 
   return {
-    visible: useSelector(getMetabotVisisble as any) as ReturnType<
-      typeof getMetabotVisisble
+    visible: useSelector(getMetabotVisible as any) as ReturnType<
+      typeof getMetabotVisible
     >,
     setVisible: useCallback(
       (isVisible: boolean) => dispatch(setVisible(isVisible)),
       [dispatch],
     ),
-    userMessages,
-    dismissUserMessage: (messageIndex: number) =>
-      dispatch(dismissUserMessage(messageIndex)),
+    messages,
+    lastAgentMessages: useSelector(
+      getLastAgentMessagesByType as any,
```

Comment:
`any` usage

Reply:
Actually, a lot of `any` in this file 😅

Reply:
Great point. This is going to have to come as a follow up task. There's several TODOs in other files related to this.